packages/server/modules/workspaces/services/projects.ts

```diff
@@ -33,3 +35,42 @@ export const queryAllWorkspaceProjectsFactory = ({
       iterationCount++
     } while (!!cursor)
   }
+
+type GetWorkspaceProjectsArgs = {
+  workspaceId: string
+}
+
+type GetWorkspaceProjectsOptions = {
+  limit: number | null
+  cursor: string | null
+  filter: UserProjectsFilter | null
```

Comment:
Even if UserProjectsFilter has the same values you want right now, you shouldn't use it and should introduce your own type cause the semantics are different. The types being the same right now doesn't mean they're gonna be the same in the future. In the future the type may get other more user specific fields added to it, and then your GetWorkspaceProjects field is gonna ask for user-specific filtering fields.

Reply:
roger, done